commit ec0994e48ea2aebf62ff08376227f3a9ccf46262
Author: Sage Weil <sage@newdream.net>
Date:   Tue Feb 2 16:25:35 2010 -0800

    ceph: add support for auth_x authentication protocol
    
    The auth_x protocol implements support for a kerberos-like mutual
    authentication infrastructure used by Ceph.  We do not simply use vanilla
    kerberos because of scalability and performance issues when dealing with
    a large cluster of nodes providing a single logical service.
    
    Auth_x provides mutual authentication of client and server and protects
    against replay and man in the middle attacks.  It does not encrypt
    the full session over the wire, however, so data payload may still be
    snooped.
    
    Signed-off-by: Yehuda Sadeh <yehuda@hq.newdream.net>
    Signed-off-by: Sage Weil <sage@newdream.net>

diff --git a/fs/ceph/Makefile b/fs/ceph/Makefile
index 85a588e43e7d..6a660e610be8 100644
--- a/fs/ceph/Makefile
+++ b/fs/ceph/Makefile
@@ -15,6 +15,7 @@ ceph-objs := super.o inode.o dir.o file.o addr.o ioctl.o \
 	debugfs.o \
 	auth.o auth_none.o \
 	crypto.o armor.o \
+	auth_x.o \
 	ceph_fs.o ceph_strings.o ceph_hash.o ceph_frag.o
 
 else